package.json

```diff
@@ -45,9 +45,6 @@
     "cypress": "npx cypress open",
     "lint": "prettier --write '{src,test}/**/*.{tsx,css}'"
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
```

Comment:
This had to be removed in order to fix an import error for es-lint. AFAIK this isn't needed anymore.